source/Calamari.AzureWebApp.NetCoreShim/Retry/RetryTracker.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
```

Comment:
Any value commenting that these are duplicated from the common.plumbing.retry?

Reply:
done